### PR TITLE
[BUG] Fix errors in the Event filters topic

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2980.

Made the following corrections to the Event filters topic:
- [Event filters](https://security-docs_2982.docs-preview.app.elstc.co/guide/en/security/master/event-filters-api.html): Changed the `list_id` value to `endpoint_event_filters` by updating the `list_id` attribute. Changes were applied across various reference tables and examples.
- [Create event filter container](https://security-docs_2982.docs-preview.app.elstc.co/guide/en/security/master/event-filters-api.html#_create_event_filters_container): Changed `type` field value from `endpoints` to `endpoint_events`.
- [Create event filter](https://security-docs_2982.docs-preview.app.elstc.co/guide/en/security/master/event-filters-api.html#create-event-filter-item-api): Updated the description for the `os_types` field to indicate that only one OS could be selected.

### CODE REVIEW

**Comment by dasansol92** [comment]
Hey @nastasha-solomon , the `type` for event filters needs to be changed also in the example and in the fields table.

Thanks for fixing this!

**Comment by dasansol92** [comment]
Hi @nastasha-solomon , the `type` in the Event Filters container table is `endpoint_host_isolation_exceptions` and it should be `endpoint_events` instead. Thanks!

**Comment by nastasha-solomon** [comment]
@dasansol92 @kevinlog it looks like the table in the [Create event filters container](https://security-docs_2982.docs-preview.app.elstc.co/guide/en/security/master/event-filters-api.html#_create_event_filters_container) and [Create host isolation exceptions container](https://security-docs_2982.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions-api.html#_create_host_isolation_exceptions_container) docs reference [the same tagged section](https://github.com/elastic/security-docs/pull/2982/files#diff-d692e1c82fad400f2605747303488a205b1a2f01aee33df1f25f4125f09cb394L54) in the `_exceptions-api-reusable-content.asciidoc` file. Because of this, if I make any changes to the referenced section, the updates will show up in both docs. 

I'm not sure whether this setup was intentional, but if the priority is to fix the Event filters API docs, I can fix the issue @dasansol92 mentioned [here](https://github.com/elastic/security-docs/pull/2982#issuecomment-1433296182) and then open a separate issue/PR for addressing the error in the Create host isolation exceptions container docs. This will allow us to merge the corrections to the Event filters API docs and handle the duplicate reference at a later time. WDYT?

**Comment by kevinlog** [comment]
> I'm not sure whether this setup was intentional, but if the priority is to fix the Event filters API docs, I can fix the issue @dasansol92 mentioned https://github.com/elastic/security-docs/pull/2982#issuecomment-1433296182 and then open a separate issue/PR for addressing the error in the Create host isolation exceptions container docs. This will allow us to merge the corrections to the Event filters API docs and handle the duplicate reference at a later time. WDYT?

@nastasha-solomon this makes sense to me

**Review by joepeeples — APPROVED** [review]
LGTM, thanks!

File: docs/management/api/_exceptions-api-reusable-content.asciidoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ created when it is not provided.
 |`meta` |Object |Placeholder for metadata about the exception item. |No
 |`name` |String |The {endpoint-artifact-name-single} name. |Yes
 |`namespace_type` |String a|Must be set to `agnostic` for endpoint artifacts. |Yes
-|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Array of xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type] values. | Yes
+|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Use this field to specify the xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[operating system]. | Yes
```
Comment (kevinlog) [inline docs/management/api/_exceptions-api-reusable-content.asciidoc:86]: Can we say here:

```
Use this field to specify the xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[operating system].  You can use only one value.
```

Just to make it clear that you can only use one OS value

**Review by kevinlog — APPROVED** [review]
Left a comment suggestion.  After that, LGTM

File: docs/management/api/_exceptions-api-reusable-content.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [reply docs/management/api/_exceptions-api-reusable-content.asciidoc:86]:
```suggestion
|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Use this field to specify the xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[operating system]. Only enter one value.  | Yes
```

**Pull request merged**